### ISSUE TITLE
"Regulatoryreview" status isn't handled as expected

### ISSUE THREAD

**Issue by grigorijurasov** [woocommerce/woocommerce-gateway-paypal-express-checkout#492]
**Reported in**

hc - 7681997

**What was expected**

When the payment has "Regulatoryreview" status in PayPal but is processed afterwards - I was hoping that the order status will change to "Processing" or "Completed" in WooCommerce

**What happened instead**

Order status stays "On hold" even after the payment is processed in PayPal

**How we found out**

I'm not aware of a way to get the payment to end up with "regulatoryreview" status in PayPal on demand but looking here https://developer.paypal.com/docs/classic/api/merchant/DoExpressCheckoutPayment_API_Operation_NVP/ there's such a status mentioned:

![screenshot](http://cld.wthms.co/hEzcDt+) **Image Link:** http://cld.wthms.co/hEzcDt

We weren't able to find the code that would be responsible for handling such status.

For our customer this always means that the payment status won't change after the payment is confirmed in PayPal.

This is perhaps an enhancement request, asked the user to also add this to our ideas board.

**Comment by allendav** [comment]
@kloon - any intersection with order status here?

**Comment by kloon** [comment]
@allendav yes, since WooCommerce currently does not have a status to handle something like pending authorization it will require a custom status to make it work with the current status implementation.

With the new payment statuses we are aiming to introduce a awaiting authorization status that can be used for this.

Conclusion if you implement this with the current statuses you will require a custom status which will likely make the migration a bit more complex for this extension.
